appdev/platform/guides/Fairphone-2-Developer-Information.rst

```diff
@@ -0,0 +1,95 @@
+Fairphone 2 Developer Information
+=================================
+Overview
+--------
+The Fairphone 2 (FP2), released in December of 2015, shipped with Android 5.1 Lollipop. The current Cyanogenmod base should be sufficient to get it stable.
+
+This page contains the information that you'll need to develop on the Fairphone 2. This includes instructions to get the source tree, patches, and changes to make after building.
+
+Development repositories for the Fairphone 2 are located on the ubports GitHub. You may submit pull requests against this source and they will be reviewed by UBPorts developers. You might want to give us a heads-up in the UBPorts IRC as well.
+
+Please remember to make a complete backup of your phone before doing any flashing operations, assuming you have anything that you want to keep on it. Then, copy the backup to your computer. Your phone may be wiped during this procedure.
+Development status
+
+Current devel rc proposed builds:
+GUI -> Working
+Wifi -> Working
+Audio -> Working
+Camera -> Working
+Bluetooth -> Working
+GSM -> Working
+Motion sensor ->Working
+GPS -> WIP
+
+Current issues
+~~~~~~~~~~~~~~
+
+GPS is really slow
+Battery life could be improved
+
+Building
+--------
+Install Prerequisites
+~~~~~~~~~~~~~~~~~~~~~
+::
+
+$ sudo apt-get install git gnupg flex bison gperf build-essential zip bzr curl libc6-dev libncurses5-dev:i386 x11proto-core-dev libx11-dev:i386 libreadline6-dev:i386 libgl1-mesa-glx:i386 libgl1-mesa-dev g++-multilib tofrodos python-markdown libxml2-utils xsltproc zlib1g-dev:i386 schedtool g++-4.8-multilib phablet-tools
+
+Set up the tree
+~~~~~~~~~~~~~~~
+
+Start by creating yourself a development directory, then move into it. Run 
+::
+
+$ repo init -u https://github.com/ubports/android -b ubp-5.1 --depth=1
+
+in this directory.
+
+Next, ``mkdir .repo/local_manifests``, then create the file ``.repo/local_manifests/fp2.xml``. 
+
+Paste the following into the file:
+
+::
+	
+	<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't think this manifest is correct. This is my fault as I didn't keep those docs up to date.